plugin/role_set.go

```diff
@@ -415,6 +416,18 @@ func roleSetServiceAccountName(rsName string) (name string) {
 	return name
 }
 
+func roleSetServiceAccountDisplayName(name string) string {
+	hashLen := 8
```

Comment:
Can we move this into the const declarations up top?

Reply:
fixed